.circleci/config.yml

```diff
@@ -63,20 +63,17 @@ commands:
       - aws-s3/copy:
           from: 's3://appcues-mobile-certs/appcues.keystore'
           to: './'
-      - aws-s3/copy:
-          from: 's3://appcues-mobile-certs/appcues-kotlin-example-google-services.json'
-          to: './samples/kotlin-android-app/google-services.json'
       - run:
           name: Generate Gradle cache key
           command: ./checksum.sh /tmp/checksum.txt
       - restore_cache:
           name: Restore Gradle cache
-          key: gradle-{{ checksum "/tmp/checksum.txt" }}
+          key: 1-gradle-{{ checksum "/tmp/checksum.txt" }}
```

Comment:
the prefix here ended up probably not needed, but also harmless - no issue if kept in I think.

Reply:
i was considering undoing it, but since we already burned the 1-* i think we need to keep now so in the future if we think its cache issue we go for 2-*